src/app/app.component.ts

```diff
@@ -14,12 +18,26 @@ import { Outlet, Router } from '@tanstack/angular-router';
     <hr />
 
     <outlet />
+    
+    <ng-container *ngIf="routerInstance">
```

Comment:
Control flow is built-in

```suggestion
   @if(routerInstance) {
```